### PR TITLE
Add copy to clipboard tag

### PR DESCRIPTION
**Pull request by UnderscoreTud**
### Description
This PR adds a copy to clipboard tag to Skript's chat components, this tag has been missing from the tags that Skript supports for a while now, and I don't see any reason on why that is.
Here's an example of how to use it:
```vb
send "<copy:text to copy>text to display"
```

---
**Target Minecraft Versions:** 1.15+
**Requirements:** none
**Related Issues:** https://github.com/SkriptLang/Skript/issues/5351

### CODE REVIEW

File: src/main/resources/lang/default.lang [hunk 1]
```diff
@@ -405,6 +405,7 @@ chat styles:
 	run_command: run command, command, cmd
 	suggest_command: suggest command, sgt
 	change_page: change page
+	copy_to_clipboard: copy to clipboard, copy, clipboard
```
Comment (Fusezion) [inline src/main/resources/lang/default.lang:408]: minor but could we add a `ctp` option as well? tince tooltip has `ttp` I figured it makes ense

Comment (Pikachu920) [reply src/main/resources/lang/default.lang:408]: I feel like copy is a better shorthand. `ctp` is confusing to me

File: src/main/java/ch/njol/skript/util/chat/SkriptChatCode.java [hunk 2]
```diff
@@ -125,6 +125,14 @@ public void updateComponent(MessageComponent component, String param) {
 		}
 	},
 	
+	copy_to_clipboard(true) {
+		@Override
+		public void updateComponent(MessageComponent component, String param) {
+			ClickEvent e = new ClickEvent(ClickEvent.Action.copy_to_clipboard, param);
+			component.clickEvent = e;
```
Comment (AyhamAl-Ali) [inline src/main/java/ch/njol/skript/util/chat/SkriptChatCode.java:132]:
```suggestion
			ClickEvent event = new ClickEvent(ClickEvent.Action.copy_to_clipboard, param);
			component.clickEvent = event;
```

**Review by AyhamAl-Ali — CHANGES_REQUESTED** [review]
Nice PR ⚡

**Review by AyhamAl-Ali — APPROVED** [review]

**Review by TheLimeGlass — CHANGES_REQUESTED** [review]
Needs to have an update to the documentation at docs/text.html

**Comment by AyhamAl-Ali** [comment]
> Needs to have an update to the documentation at docs/text.html

This will have to be in the other repo https://github.com/SkriptLang/skript-docs/ so this should be good to go, another PR should be opened there either parallel or after both are okay

**Review by TheLimeGlass — APPROVED** [review]

**Pull request merged**